.github/workflows/neo4j-aura-resume.yml

```
@@ -0,0 +1,69 @@
+        uses: actions/checkout@v4
+
+      - name: Resume paused Aura instance
+        id: resume
+        continue-on-error: true
+        run: |
+          set -euo pipefail
+          python3 scripts/resume_neo4j_aura.py
+
+      - name: Notify deployment analytics when Aura was resumed
+        if: ${{ steps.resume.outcome == 'success' && steps.resume.outputs.action == 'resumed' }}
+        uses: ./.github/actions/send-slack-notification
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/send-slack-notification": line 5: unexpected key "type" for definition of input "verbiage"

(action)